heartbeat/galera

```diff
@@ -955,7 +966,7 @@ fi
 : ${OCF_RESKEY_check_user="root"}
 
 MYSQL_OPTIONS_CHECK="-nNE --user=${OCF_RESKEY_check_user}"
-if [ -n "${OCF_RESKEY_check_passwd}" ]; then
+if [ -n "${OCF_RESKEY_check_passwd}" -o \( -z "${OCF_RESKEY_check_passwd}" -a "${OCF_RESKEY_check_passwd_use_empty}" = 1 \) ]; then
```

Comment:
I think it'd be better to do:
`if ! ocf_is_true "${OCF_RESKEY_check_passwd_use_empty}" && [ -n "${OCF_RESKEY_check_passwd}" ]; then`
here, and undo the other if-change above that wont be needed.

ocf_is_true returns true/false for 1/0, true/false, yes/no values.

Reply:
So if `check_passwd` is set **and** `check_passwd_use_empty` is `1 (or yes or true)`, `check_passwd_use_empty` should have precedence? Current implementation is the other way arround...

Reply:
Presedence doesnt matter when it's only an AND.

Reply:
But your check seems to be wrong. I updated the code and I think that should be ok now.